### ISSUE TITLE
ListView incorrectly intercepts PointerRelease for selection without having PointerPressed

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#843]
## I'm submitting a...
- Bug report

## Current behavior
Clicking outside of a ListView, holding, then releasing inside of a listview changes the selected item.

## Expected behavior
Releasing the pointer should not change the selection

## Environment

```
Nuget Package: 1.45.0-dev.1359

Affected platform(s):
- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

**Comment by davidjohnoliver** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [WASM ListView pointer intercept. https://github.com/unoplatform/uno/issues/843](https://trello.com/c/IeQP6RCO/34-wasm-listview-pointer-intercept-https-githubcom-unoplatform-uno-issues-843)

**Comment by davidjohnoliver** [comment]
In fact all controls are currently affected, the same occurs for `Button.Click` (not in a list).

**Comment by davidjohnoliver** [comment]
@dr1rrb Is this issue fixed?

**Comment by dr1rrb** [comment]
Yes fixed by #1256 🎉
